src/dist/replication/lib/replica_stub.cpp

```diff
@@ -902,6 +902,83 @@ void replica_stub::on_query_replica_info(const query_replica_info_request &req,
     resp.err = ERR_OK;
 }
 
+void replica_stub::on_query_disk_info(const query_disk_info_request &req,
```

Comment:
I suggest add comment for this function to show it executes in default thread pool, like this:
`// ThreadPool: THREAD_POOL_DEFAULT`